team_production_system/models/notification.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
```

Comment:
Pre-existing linting error:
Needs one more blank line between imports and class (comment descriptions do not apply)